ipwb/replay.py

```diff
@@ -276,6 +276,11 @@ def getCDXJLinesWithURIR(urir, indexPath):
         indexPath = ipwbUtils.getIPWBReplayIndexPath()
     indexPath = getIndexFileFullPath(indexPath)
 
+    # Check that file exists prior to executing operations on it
+    if not os.path.isfile(indexPath):
```

Comment:
This check is performed in the wrong place. This function is being called later on various lookup requests. The existence of the index file should be checked at the time of [starting replay](https://github.com/oduwsdl/ipwb/blob/a6cecf1e9ae592e84120819a819ccf0beebd0bf6/ipwb/replay.py#L973) and should exit if the index is not found.